test/src.js

```diff
@@ -128,35 +131,35 @@ describe('.src()', function() {
 
     srcStream.write(file);
 
-    pipe([
+    pipeline([
       srcStream,
       concat(assert),
     ], done);
   });
 
   it('removes BOM from utf8-encoded files by default (buffer)', function(done) {
-    var expectedContent = new Buffer(bomContents.replace('X', '8'));
+    var expectedContent = Buffer.from(bomContents.replace('X', '8'));
 
     function assert(files) {
       expect(files.length).toEqual(1);
-      expect(files[0].contents).toMatch(expectedContent);
+      expect(files[0].contents.toString()).toEqual(expectedContent.toString());
```

Comment:
Need to remove these toString calls. Noting to remove when I refactor this file.

Reply:
I removed `.toString()` here and others in `test/src.js`.
But some places were not removed because data types of them are object and they caused errors by removing.

Reply:
The tests that were failing seem to be due to an incorrect `concat` implementation. I refactored the stream utils a bit.